### PR TITLE
changes to episode 4, tasks 28-29

### PR DESCRIPTION
**Pull request by mallewellyn**
Largely trying to introduce consistent package use throughout as in #117 - this version only uses PCAtools and provides the table for conversion to base R PCA functions.

I have also introduced a consistent scree plot definition (variance explained rather than cumulative) in line with what I believe to be the standard definition - though absolutely feel free to correct me! PCAtools plots both, which is informative, but possibly not a scree plot and therefore confusing.

This will be ugly to merge with the other pull requests related to this issue but happy to work my way through these as we go.

### CODE REVIEW

**Comment by mallewellyn** [comment]
Episode needs to be checked once all PRs combined (and rendered)

File: _episodes_rmd/04-principal-component-analysis.Rmd [hunk 1]
```diff
@@ -556,7 +560,7 @@ representing groups of genes would help visualise the data and address
 research questions regarding the effect different groups of genes have on
 disease progression.
 
-Using the **`PCAtools`** we will apply a PCA to the cancer
+Using the **`PCAtools`**, we will apply a PCA to the cancer
```
Comment (alanocallaghan) [inline _episodes_rmd/04-principal-component-analysis.Rmd:563]: Using the **`PCAtools`**, -> Using `PCAtools`,

Comment (alanocallaghan) [reply _episodes_rmd/04-principal-component-analysis.Rmd:563]: Also phrasing of "we will apply a PCA to". I'd maybe say "we will perform PCA on" or in casual language "run PCA on"

Comment (mallewellyn) [reply _episodes_rmd/04-principal-component-analysis.Rmd]: Removed bold PCA tools in [fd812a7](https://github.com/mallewellyn/high-dimensional-stats-r/commit/fd812a7e37211492176ab28958ef4163ff3de3cc)

Comment (mallewellyn) [reply _episodes_rmd/04-principal-component-analysis.Rmd]: Hang on... did I want to do that? Going through the episodes, some packages are in bold and some are not. Can change all, but which do you prefer?

Comment (mallewellyn) [reply _episodes_rmd/04-principal-component-analysis.Rmd]: Both the wording changes above in 5ab22d2

Comment (alanocallaghan) [reply _episodes_rmd/04-principal-component-analysis.Rmd]: Hmm you're right, let's go with bold for all, apologies

Comment (mallewellyn) [reply _episodes_rmd/04-principal-component-analysis.Rmd]: In [5844f4b](https://github.com/mallewellyn/high-dimensional-stats-r/commit/5844f4b533a74eb9a15baf3debd95b31745842e3)

**Review by alanocallaghan — APPROVED** [review]

**Pull request merged**